pkg/network/node/ovscontroller.go

```diff
@@ -53,18 +54,25 @@ func (oc *ovsController) getVersionNote() string {
 }
 
 func (oc *ovsController) AlreadySetUp(vxlanPort uint32) bool {
-	flows, err := oc.ovs.DumpFlows("table=%d", ruleVersionTable)
-	if err != nil || len(flows) != 1 {
+	ruleVersionFlows, err := oc.ovs.DumpFlows("table=%d", ruleVersionTable)
+	if err != nil || len(ruleVersionFlows) != 1 {
+		return false
+	}
+	// Have a look in the table filled in by setupPodFlows, as to make
+	// sure that we don't have a partial setup remaining in other tables,
+	// but are actually missing pod setup from an earlier run and don't
+	// proceed to setting up any pods that we should be doing in this run.
+	podFlows, err := oc.ovs.DumpFlows("table=%d", podFlowsTable)
+	if err != nil || len(podFlows) == 0 {
```

Comment:
considering the many pods each worker will have on openshift, I guess this condition to check for at least one podFlow is ok, as in there will anyways never be a condition where we don't have at least one non-hostnetwork pod on the node.

Reply:
> as in there will anyways never be a condition where we don't have at least one non-hostnetwork pod on the node.

Well, there will be when we start fresh, but in that case we want to set everything up anyways. I guess I could have re-worked the code to check that each individual pod that is returned in `GetPodNetworkInfo` is passed to `alreadySetup` and check each one...but I figured that reworks the code too much and might lead to more entropy and more cases of "full setup".

Reply:
If there are actually no (pod-network) pods then there's not really much downside to destroying and recreating the bridge anyway